templates/rap_sheet_instructions.jinja

```diff
@@ -13,139 +13,260 @@
+      <p>{{ _("Some legal aid organizations, like the East Bay Community Law Center, "
+          "need your California criminal record (RAP sheet) to check what services you "
+          "qualify for. Follow the steps below to get your RAP sheet and send it to "
+          "East Bay Community Law Center to complete your Alameda County application.") }}
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="rap_fee_waiver">
+  <div class="row">
+    <div class="col-xs-12">
+      <h2>{{ _(" You qualify to get $25 off of your RAP sheet") }}</h2>
+      <p>{{ _("You qualify for a fee waiver. This means that the California Department of Justice "
+            "will cover $25 of your RAP sheet fees. Even though you will get $25 off your RAP "
+            "sheet fees, you will still have to pay the rest. The fee you will have to pay is "
+            "usually between $20 and $50. ") }}
+      </p>
+      <p>
+      {{ _(" **Applying for a fee waiver could add 2-4 weeks to the process of clearing "
+        "your record and is optional. If you would not like to wait 2-4 weeks to get $25 off "
+        "of your RAP sheet, you can skip to the next section on how to get your RAP sheet.") }}
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="rap_fee_waiver_instructions">
+  <div class="row">
+    <div class="col-xs-12">
+      <h4>
+        <span class="glyphicon glyphicon-circle-arrow-right" aria-hidden="true"></span>
+        {{ _(" Follow these steps to get $25 off of your RAP sheet fees:") }}
+      </h4>
+      <ol class="rap_sheet_instructions">
+        <li>
+          {{ _('Print and fill out this form with your information: '
+          '<a href="https://drive.google.com/open?id=0B3BLnlbsdQ_hQUlELWJKT1hKRVE">Fee Waiver Application Form</a> ') }}
+        </li>
+        <li>
+          {{ _("If you are receiving any government benefits, make a copy of those documents. "
+            "Here are some government benefits that will work: ") }}
+            <ul class="rap_instructions_details">
+              <li class="rap_detail">
+                {{ _("Supplemental Security Income (SSI) ") }}
+              </li>
+              <li class="rap_detail">
+                {{ _("California Work Opportunity and Responsibility to Kids (CalWORKs)  ") }}
+              </li>
+              <li class="rap_detail">
+                {{ _("Unemployment benefits ") }}
+              </li>
+              <li class="rap_detail">
+                {{ _("Disability insurance ") }}
+              </li>
+              <li class="rap_detail">
+                {{ _("EBT benefits card  ") }}
+              </li>
+            </ul>
+        </li>
+        <li>
+          {{ _("Mail your completed Fee Waiver Application form and copies of your government "
+            "benefits (if you are receiving benefits) to this address: ") }}
+            <pre class="address">
+      California Department of Justice
```

Comment:
This address is also formatted using `pre` tag. If we change address formatting, should be consistent across the board.